chanacceptor/rpcacceptor.go

```diff
@@ -0,0 +1,198 @@
+package chanacceptor
+
+import (
+	"errors"
+	"sync"
+	"sync/atomic"
+	"time"
+)
+
+const (
+	// DefaultRPCTimeout is the default time that the Accept method will
+	// listen for responses from the RPC server, after which it will timeout.
+	DefaultRPCTimeout = 15 * time.Second
+)
+
+// ErrNoChainedAcceptor is returned when the RPCAcceptor is unable to retrieve
+// the ChainedAcceptor singleton.
+var ErrNoChainedAcceptor = errors.New("unable to get chained acceptor")
+
+// OpenChannelResponse is a struct containing the response from the RPC client
```

Comment:
I went with `OpenChannelResponse` since it mirrors `OpenChannelRequest`, but I'm totally open to changing it.  I think `ChannelAcceptResponse` is a better name but it doesn't like implicitly convey that it's the `OpenChannelRequest`s opposite if you catch my drift.

Reply:
Reason I don't want to use `OpenChannelRequest`/`Response` is that we already have `OpenChannelRequest` and `openChanReq` in used in the codebase iirc 😛